File: HMSPADemo2.Domain/Entities/OrderDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HMSPADemo2.Domain.Entities
{
    public class OrderDetails
    {
        public int Id { get; set; }
        public string Description { get; set; }
        public decimal TotalCost { get; set; }
    }
}

File: HMSPADemo2.Web/app/orderdetails/entities/OrderDetails.ts
module app.orderdetails {
    'use strict';

    export interface IOrderDetails {
        id: number;
        description: string;
        totalCost: number;
    }

    export class OrderDetails implements IOrderDetails {
        id: number;
        description: string;
        totalCost: number;
    }
}